source/standards/optimise-frontend-perf.html.md.erb

```diff
@@ -0,0 +1,103 @@
+---
+title: How to optimise frontend performance
+expires: 2018-12-06
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+You should focus on [frontend performance][] when developing your service’s website. This will improve the user experience of your service by making your website respond faster and work better on all devices.
+
+## Prioritise performance tasks
+
+You can optimise your site’s frontend performance by prioritising tasks that will improve your site speed. Prioritise things you must do (high) over medium or low priority (nice to have) tasks.
+
+For example:
+
+|Priority|Example|Action
+|---|---|---|
+|High|Position styles correctly|Set styles at the top and scripts at the bottom of a webpage|
+||Minimise HTTP requests|Minimise tiling icons, CSS and JavaScript files to reduce size and loading time|
+||Compress static resources|Use [minification][] or [GNU Gzip][] to compress CSS and JavaScript code|
+||Set correct Headers|Set correct [Cache-Control][] and [ETag][] headers on assets for optimal caching|
+|Medium|Look for empty image `src` attributes|Avoid using empty image `src` attributes as some browsers always send requests to them, resulting in additional traffic|
+||Minimise DNS lookups|Use fewer domains to reduce the number of DNS lookups per page|
```

Comment:
I think this should say "third-party domains" else it could be seen as a contradiction to the point below.